lib/index.js

```diff
@@ -168,12 +171,19 @@ class PluginPassportOAuth {
     return this.getProviderRepository(profile.provider)
       .get(profile._json[this.config.strategies[profile.provider].identifierAttribute])
       .then(userObject => {
-        let req = null;
-
         if (userObject !== null) {
           return Promise.resolve({kuid: userObject.kuid, message: null});
         }
 
+        return Promise.resolve({kuid: null, message: `Could not login with strategy "${profile.provider}"`});
```

Comment:
I wonder if this line of code is still reachable...
I mean: if `getProviderRepository` resolves, could the result be `null` ?

Reply:
I was asking myself that too and I thought we are never too careful. But yes rejecting with a null kuid seems better